xbmc/guilib/GUIControl.cpp

```diff
@@ -951,6 +955,25 @@ void CGUIControl::UpdateControlStats()
   }
 }
 
+bool CGUIControl::IsControlRenderable()
```

Comment:
its possible add an explanation comment somewhere?
when a new GUI control should be added in this list?

Reply:
Done.

It is needed as textures might be rendered outside of the bounding box of a control.

Reply:
a group culling its children would be a different type of culling I guess, at a higher level, nvm!